src/Microsoft.Framework.PackageManager/Utils/FileOperationUtils.cs

```diff
@@ -38,12 +37,6 @@ private static void DeleteRecursive(string deletePath)
 
         public static bool MarkExecutable(string file)
         {
-            if (PlatformHelper.IsWindows)
```

Comment:
Sounds good. This is a shared util method and putting OS check in it makes code DRYer.